README.md

```diff
@@ -242,10 +243,22 @@ ignored:
 Ignored files/folders can be specified via standard pattern-matching.
 Hidden files (dotfiles, i.e. `.git`, `.gitignore`, etc) are ignored by default.
 
+# Custom Todos
+You can customize your set of variables to be considered as todos. Specify it in the `custom_todos` section in the `.todocheck.yaml`:
+```
+origin: some.origin.com
+issue_tracker: JIRA
+custom_todos:
+  - "@fix"
+  - ToDo
+```
+
```

Comment:
Let's also specify an example usage:

With the above configuration, `todocheck` will not start matching todo comments in the following formats:
```
// @fix #234: a leftover task...
// ToDo #15: A funky-looking todo comment
```